src/Components/Blazor/Validation/src/ComparePropertyAttribute.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.ComponentModel.DataAnnotations
+{
+    /// <summary>
+    /// A <see cref="ValidationAttribute"/> that compares two properties
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+    public sealed class ComparePropertyAttribute : CompareAttribute
```

Comment:
👍

Reply:
This class should be in the System.ComponentModel.DataAnnotations project, not Blazor.

To add this validation to a property in `MyApp.Shared` means the shared app has to reference the Blazor assemblies - which isn't good for a contracts assembly that could be used in different types of client apps.